### PR TITLE
:sparkles: Allow moderator/admin to take down user account

### PR DESCRIPTION
**Pull request by foysalit**

### CODE REVIEW

File: packages/pds/src/api/com/atproto/admin/takeModerationAction.ts [hunk 1]
```diff
@@ -28,9 +28,9 @@ export default function (server: Server, ctx: AppContext) {
       } = input.body
 
       // apply access rules
-
+      const canActorTakedown = access.admin || access.moderator
       // if less than admin access then can not takedown an account
-      if (!access.admin && action === TAKEDOWN && 'did' in subject) {
+      if (!canActorTakedown && action === TAKEDOWN && 'did' in subject) {
         throw new AuthRequiredError(
           'Must be an admin to perform an account takedown',
```
Comment (mletterle) [inline packages/pds/src/api/com/atproto/admin/takeModerationAction.ts:35]: Should be "Must be an admin or moderator to perform an account takedown"?

**Comment by stella3d** [comment]
lol

**Comment by stella3d** [comment]
please don't ever merge this or anything like it thanks

**Comment by devinivy** [comment]
@foysalit thanks for hopping on this!  Ended-up making some final tweaks here: https://github.com/bluesky-social/atproto/pull/1330

**Pull request closed without merging**